portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js

```diff
@@ -221,11 +221,11 @@ var StudyViewProxy = (function() {
                         var _parts = _lists[i].split('\t');
```

Comment:
The structure of that object, which is based on the case lists:

0: "case_list_id"
1: "case_list_name"
2: "case_list_description"
3: "cancer_study_id"
4: "case_ids"

If the length isn't 5, there is something wrong with the case list and the sample Ids won't be able to be extracted.

I just hit the webservice for case lists with wget (http://www.cbioportal.org/webservice.do?cmd=getCaseLists&cancer_study_id=brca_tcga), and it looks like there is an extra space at the end of each case list returned. Looking in the database (sample_list_list) there are no blank entries. @jjgao should we be doing anything else for this or just trim the response from the webservice?